for_workspace/make_build.bzl

```diff
@@ -50,6 +53,7 @@ make_tool = rule(
             doc = "target with the Make sources",
             mandatory = True,
         ),
+        "_cc_toolchain": attr.label(default = Label("@bazel_tools//tools/cpp:current_cc_toolchain")),
```

Comment:
Shouldn't this just go in the list of toolchains below?

Reply:
No; see https://docs.bazel.build/versions/master/integrating-with-rules-cc.html